manifest/rfc2822.go

```diff
@@ -249,7 +322,7 @@ func Parse2822(readerIn io.Reader) (*Manifest2822, error) {
 	if invalidMaintainers := manifest.Global.InvalidMaintainers(); len(invalidMaintainers) > 0 {
 		return nil, fmt.Errorf("invalid Maintainers: %q (expected format %q)", strings.Join(invalidMaintainers, ", "), MaintainersFormat)
 	}
-	if len(manifest.Global.Tags) > 0 {
+	if len(manifest.Global.Tags) > 0 || len(manifest.Global.SharedTags) > 0 {
```

Comment:
Wouldn't global SharedTags be valid and useful if there is only one version of stuff? :thinking:

```
Maintainers: Tianon Gravi <admwiggin@gmail.com> (@tianon),
             Joseph Ferguson <yosifkit@gmail.com> (@yosifkit)
GitRepo: https://github.com/docker-library/hello-world.git
SharedTags: latest

Tags: linux
GitCommit: bdee60d7ff6b98037657dc34a10e9ca4ffd6785f
Directory: hello-world

Tags: nanoserver
GitCommit: 1f13a5bc3b787747eeefb3b0051d8d29f851260d
Directory: hello-world/nanoserver
Constraints: nanoserver
```